src/Resources/config/no_driver_admin.xml

```diff
@@ -1,3 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <!--
+     This file will be loaded when SonataAdminBundle is enabled and db_driver has it's default value, not corresponding to real persistence mechanism.
```

Comment:
- it's => its
- linebreak that line 4

 and we're good

Reply:
Fixed